composer.json

```diff
@@ -2,32 +2,50 @@
     "name": "tugboatqa/wordpress",
     "description": "Wordpress Demo",
     "type": "project",
-    "autoload": {
-        "psr-4": {
-            "Tugboatqa\\Wordpress\\": "src/"
-        }
-    },
     "authors": [
         {
             "name": "KeyboardCowboy",
             "email": "chris.albrecht@lullabot.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://wpackagist.org",
+            "only": [
+                "wpackagist-plugin/*",
+                "wpackagist-theme/*"
+            ]
+        }
+    ],
     "require": {
         "php": ">=8.0",
-        "roots/wordpress": "6.2.2"
+        "composer/installers": "^1.3.0",
+        "cweagans/composer-patches": "^1.7",
+        "roots/wordpress": "^6.3",
+        "wpackagist-plugin/advanced-custom-fields": "^6.2",
+        "wpackagist-theme/kadence": "^1.1",
+        "wp-cli/wp-cli-bundle": "^2.8"
     },
     "extra": {
-        "wordpress-install-dir": "docroot"
+        "wordpress-install-dir": "docroot/wp",
```

Comment:
Are we installing WordPress Core in a subdirectory of the `docroot` subdirectory?